miner/worker.go

```diff
@@ -192,6 +192,9 @@ func (w *worker) commitNewWork(predicateContext *precompileconfig.PredicateConte
 	}
 
 	// Fill the block with all available pending transactions.
+	if header.BaseFee != nil {
+		w.eth.TxPool().SetBaseFee(header.BaseFee)
```

Comment:
Can this basefee be overriden by update basefee interval in txpool? I wonder if we should be delaying the interval after setting this here so that the basefee set here will be used.